components/gcp/automl/create_dataset_for_tables/component.py

```diff
@@ -51,7 +51,7 @@ def automl_create_dataset_for_tables(
         region=gcp_region,
         dataset_id=dataset_id,
     )
-    return (dataset.name, dataset.create_time, dataset_id, dataset_url)
+    return (dataset.name, str(dataset.create_time), dataset_id, dataset_url)
```

Comment:
Do you mind dropping a comment about why this casting is needed? Thanks!

Reply:
While writing a comment, the @google-oss-robot merged the PR 😂 
Should I open another PR?

Reply:
My bad🤣 I forgot the auto-merge.

Feel free to do that once you have a chance. Thanks!